pkg/templates/templates/create/porter.yaml

```diff
@@ -6,10 +6,10 @@
 name: HELLO
 version: 0.1.0
 description: "An example Porter configuration"
-tag: getporter/porter-hello:latest
+tag: getporter/porter-hello:v0.1.0
 
 # Uncomment the line below to set a specific name for the invocation image
-# invocationImage: getporter/porter-hello-installer:latest
+#invocationImage: getporter/porter-hello-installer:0.1.0
```

Comment:
I would prefer that we don't put this in the default porter.yaml that we generate in every `porter create`. It is a capability that people _can do_ but isn't something common enough that we should prompt them to think about when starting a new bundle. It's enough that it's documented in case someone needs to for an edge case (honestly I'm not sure what the edge case would be, once we have a firm example that should go in the doc).

I missed this when it went in, and will create a follow-up PR to remove it.

Reply:
+1

Reply:
Oops I missed that this is a new change introduced in this PR. Would you please remove invocation image from the template and the comment prompting them to set it in this PR?

Reply:
Ok, will remove that section.

Should we add a `TODO` around updating the image registry?  This is what is currently in the `build-image.md` docs:

```
description: "An example Porter configuration"
# TODO: update the registry to your own, e.g. myregistry/porter-hello:v0.1.0
tag: getporter/porter-hello:v0.1.0
```

Reply:
Oh yes good call, let's have the TODO 💯

Reply:
TODO added